data/tlg0001/tlg001/tlg0001.tlg001.perseus-grc2.xml

```diff
@@ -1,90 +1,97 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
```

Comment:
not sure if there were changes to this work? looks like some spacing was altered

Reply:
I think my editor just did this automatically. I can remove it from the PR.